google-cloud-bigquery/lib/google/cloud/bigquery/external.rb

```diff
@@ -1224,6 +1224,52 @@ def skip_leading_rows= row_count
             frozen_check!
             @gapi.google_sheets_options.skip_leading_rows = row_count
           end
+
+          ##
+          # Range of a sheet to query from. Only used when non-empty. Typical
+          # format: `{sheet_name}!{top_left_cell_id}:{bottom_right_cell_id}`.
+          #
+          # @return [Integer]
```

Comment:
What is the integer that is returned? Can you add a description here?

Reply:
Maybe this is a c/p error and it should be `String`?

Reply:
Fixed.